tools/scalar-schema/README.md

```diff
@@ -0,0 +1,80 @@
+# Schema Tool for Scalar DB
+This tool makes schemas on Cassandra or Cosmos DB for [Scalar DB](https://github.com/scalar-labs/scalardb).
```

Comment:
```suggestion
This tool creates Scalar DB schemas on Cosmos DB and Cassandra.
```
I changed the order because the following doc always explains Cosmos DB first.

Reply:
Thank you! I fixed it.